morpho5/vm/vm.c

```diff
@@ -241,6 +241,24 @@ static void vm_bindobject(vm *v, value obj) {
 #endif
 }
 
+/** @brief Binds an object to a Virtual Machine without garbage collection.
```

Comment:
Look like this is here so we can build the varg list while in the middle of calling a function.... cool